macros/calibration/flow_calibrate.cfg

```diff
@@ -70,15 +71,15 @@ gcode:
     #
     {% set do_raft = params.DO_RAFT|default(1)|int %} # whether to print a raft or not
     {% set do_retract = params.DO_RECTRACT|default(0)|int %} # whether to enable retract/unrectract on travel moves
-    {% set print_size = params.SIZE|default(40)|int %} # size of the printed object on the build plate
+    {% set print_size = params.PRINT_SIZE|default(40)|int %} # size of the printed object on the build plate
     {% set print_height = params.HEIGHT|default(15)|int %} # height of the printed object
     {% set corner_radius = params.CORNER_RADIUS|default(8)|int %} # radius of the corners to smooth the shell and toolpath
     {% set number_of_perimeters = params.PERIMETERS|default(2)|int %} # number of perimeters to print the shell
     {% set fan_speed = params.FAN_SPEED|default(20)|int %} # part cooling fan speed in percent (0-100)
 
-    {% set e_multiplier = params.EXTRUSION_MULTIPLIER|default(0.93)|float %} # extrusion multiplier for the shell
+    {% set e_multiplier = params.EXTRUSION_MULTIPLIER|default(1.00)|float %} # extrusion multiplier for the shell
     {% set filament_diameter = params.FILAMENT_DIAMETER|default(1.75)|float %} # filament diameter
-    {% set extrusion_width = params.EXTRUSION_WIDTH|default(0.4)|float %} # extrusion width goal for one line
+    {% set extrusion_width = params.EXTRUSION_WIDTH|default(0.40)|float %} # extrusion width goal for one line
```

Comment:
I'm ok with that if you want to make it more generic with lower speed values.

But in that case, I also want to keep my own "standard" values as defaults in the` variables.cfg` file used for the `CALIBRATE` macro (as talked in the other discussion)